notebook/notebook/templates/workshop/resources.html

```diff
@@ -0,0 +1,8 @@
+{% extends "layout.html" %}
+{% block title %}Resourcse{% endblock %}
+{% block content %}
+  <div class="vcentered">
+    <h1>Workshop Resourcse</h1>
+    <p>Coming...</p>
```

Comment:
What's the plan here?  I can't tell what this is for.

Reply:
Links to slides, workshop tarballs.